packages/manager/apps/web/client/app/hosting/advices/hosting-pro/advice-hosting-pro.controller.js

```diff
@@ -0,0 +1,148 @@
+import compact from 'lodash/compact';
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $http,
+    $q,
+    $state,
+    $translate,
+    Hosting,
+    OvhApiMe,
+    RedirectionService,
```

Comment:
Use url builder instead (see previous/next comment).

Reply:
This is no longer needed, since the urls have been moved to the 2APIs